### PR TITLE
Add sudo to the shell commands

### PR DESCRIPTION
**Pull request by obadz**
..so that they can be readily pasted

### CODE REVIEW

**Comment by txdv** [comment]
Depends on the system. Easier to use doesn't mean safer to use.

**Comment by akoeplinger** [comment]
I'm a bit torn on this one. Debian doesn't have sudo by default (so users there can't copy/paste either), which means the only real argument in favor of sudo I have is that Ubuntu is the go-to distribution for users that are new to Linux, so they might not immediately know what "run this in a root shell" means.

Debian users on the other hand will more likely know that they need to remove `sudo` from commands and switch to a root shell instead.

I guess I'm slightly :+1: for taking this change, what do others think?

**Comment by obadz** [comment]
I think sudo has become the standard way to encode into publicly posted commands that they should be run as root. Ubuntu & Mint ship it by default and I think Debian has it in the standard system utilities task? (isn't that installed unless specifically unchecked by user during install?)

File: docs/getting-started/install/linux/index.md [hunk 1]
```diff
@@ -14,16 +14,16 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 Debian, Ubuntu, and derivatives
 -------------------------------
 
-Add the Mono Project GPG signing key **in a root shell** with:
+Add the Mono Project GPG signing key with (if you don't use sudo, be sure to switch to root):
```
Comment (akoeplinger) [inline docs/getting-started/install/linux/index.md]: any reason for `with`? I think it reads better without.

**Comment by akoeplinger** [comment]
Please also change the other occurences of > to | sudo tee.

**Comment by akoeplinger** [comment]
Thanks! If you have time, you could make the same changes to the MonoDevelop guide, to keep everything consistent: https://github.com/mono/md-website/blob/gh-pages/download/linux.md

**Pull request merged**